apps/login/src/lib/cookies.ts

```diff
@@ -56,13 +60,24 @@ export async function addSessionToCookie<T>(
   if (index > -1) {
     currentSessions[index] = session;
   } else {
-    currentSessions = [...currentSessions, session];
+    const temp = [...currentSessions, session];
+
+    if (JSON.stringify(temp).length >= MAX_COOKIE_SIZE) {
+      console.log("WARNING COOKIE OVERFLOW");
+      // TODO: improve cookie handling
+      // this replaces the first session (oldest) with the new one
```

Comment:
Is there an issue? Let's add the link to the comment?

Reply:
no issue.. we just have limitations on the cookie size.. so we replace the oldest session.